ZedGraph/source/ZedGraph/Scale.cs

```diff
@@ -2581,9 +2578,9 @@ internal void SetScaleMag( double min, double max, double step )
 				int numDec = 0 - (int) ( Math.Floor( Math.Log10( _majorStep ) ) - _mag );
 				if ( numDec < 0 )
 					numDec = 0;
-				_format = "f" + numDec.ToString();
```

Comment:
I had to change the format string because the old one could only go to "f99" which stopped us from properly displaying values smaller than 1e-99. The new one doesn't have that limitation while the behaviour should be the same. See the fiddle I made:
https://dotnetfiddle.net/2yILXm